.coveralls.yml

```diff
@@ -1 +0,0 @@
-src_dir: .
```

Comment:
do we still need this file?

Reply:
No, I removed it and Travis still passes. Thank you!